app/jobs/regular/sync_jira_project.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module ::Jobs
+  class SyncJiraProject < ::Jobs::Base
+    class RateLimited < ::StandardError
+    end
+
+    sidekiq_options queue: "low"
+
+    sidekiq_retry_in do |count, exception|
+      if exception.is_a?(RateLimited)
+        retry_after = exception.message.to_i
+        rand(retry_after..(2 * retry_after))
```

Comment:
Hmm is there a chance the `response["Retry-After"]` returns a value such that `to_i` results in a zero-value. `2*0 = 0`